yt_dlp/extractor/radiko.py

```diff
@@ -243,6 +246,7 @@ def _real_extract(self, url):
         return {
             'id': station,
             'title': title,
+            'artist': performer,
```

Comment:
note to maintainers: after #8917 is merged, this should be `artists` with a list value. until then leave it as `artist` w/ string value